subsys/bluetooth/CMakeLists.txt

```diff
@@ -4,7 +4,12 @@ target_include_directories(subsys__bluetooth INTERFACE ${CMAKE_CURRENT_SOURCE_DI
 
 add_subdirectory(common)
 add_subdirectory_ifdef(CONFIG_BT_HCI host)
-add_subdirectory_ifdef(CONFIG_BT_CTLR controller)
 add_subdirectory_ifdef(CONFIG_BT_SHELL shell)
 
+if(CONFIG_BT_CTLR)
+  if(CONFIG_BT_LL_SW)
+    add_subdirectory(controller)
```

Comment:
so we've decided that we are not going to place the hci drivers for binary controllers in `controller/` then right?

Reply:
Regardless of the decision, we all agree that it's not going to be in `controller/`, so this is fine